en/log/flight_review.md

```diff
@@ -0,0 +1,332 @@
+- Mouse scrolling inside the plot zooms both axes.
+
+## PID Tracking Performance {#tracking}
+
+Depending on the flight mode, the vehicle controllers may attempt to track position, velocity, altitude or rate setpoints (the tracked setpoints depend on the mode, e.g.: in Stabilized mode there is no velocity setpoint).
+
+The **Estimated** line (red) should closely match with the **Setpoint** (green).
+If they do not, in most cases the PID gains of that controller need to be tuned.
+
+The [Multicopter PID Tuning Guide](https://docs.px4.io/en/config_mc/pid_tuning_guide_multicopter.html) contains example plots and information about analysing tracking performance.
+
+> **Tip** For the rate controller in particular, it is useful to enable the high-rate logging profile ([SDLOG_PROFILE](../advanced/parameter_reference.md#SDLOG_PROFILE)) to get more details when zooming in.
+
+
+## Vibration
+
+Vibration is one of the most common problems for multirotor vehicles.
+High vibration levels can lead to:
+- less efficient flight and reduced flight time
+- the motors can heat up
+- increased material wearout
+- inability to tune the vehicle tightly, resulting in degraded flight performance.
+- sensor clipping
+- position estimation failures, potentially resulting in fly-aways.
+
+It is therefore important to keep an eye on the vibration levels and improve the setup if needed.
+
+There is a point where vibration levels are clearly too high, and generally lower vibration levels are better. 
+However there is a broad range between 'everything is ok' and 'the levels are too high'.
+This range depends on a number of factors, including vehicle size - as larger vehicles have higher inertia, allowing for more software filtering (at the same time the vibrations on larger vehicles are of lower frequency).
+
+The following paragraphs and sections provide information about what plots to use for checking vibration levels, and how to analyse them.
+
+> **Tip** It is worth looking at multiple charts when analysing vibration (different charts can better highlight some issues).
+
+
+### Actuator Controls FFT
+
+> **Note** You need to enable the high-rate logging profile ([SDLOG_PROFILE](../advanced/parameter_reference.md#SDLOG_PROFILE)) to see this plot.
+
+This graph shows a frequency plot for the roll, pitch and yaw axis based on the actuator controls signal (the PID output from the rate controller). 
+It helps to identify frequency peaks and configuring the software filters.
+There should only be a single peak at the lowest end up to a few Hz, the rest should be low and flat.
+
+Note that the y-axis scaling is different for different vehicles, but logs from the same vehicle can be directly compared to each other.
+
+#### Examples: Good Vibration
+
+[QAV-R 5" Racer](https://docs.px4.io/en/frames_multicopter/qav_r_5_kiss_esc_racer.html) frame (excellent vibration).
+Note that this allows us to considerably increase the cutoff frequency of the software filters
```

Comment:
This comment was in original version. I have just put it on first mention. 

What does this mean? When do we have to increase the frequency and when decrease it. I would have assumed we would want to decrease the frequency cutoff to the lowest frequency reasonable. We could also put the param that does this here as a link.

Reply:
Thanks for the explanation. I can see that we want to increase the filter cutoff frequency to reduce the control latency. We don't want to increase too far because this results in more noise.

But the statement is still confusing. See below - what is it about  "THIS" plot that means we can increase the cutoff frequency? Is it just that the initial bump is slow low, or is that the rest is flat, or that the signal isn't noisy?

> Note that **this** allows us to considerably increase the cutoff frequency of the software filters